File: weather-app/src/main/java/rcgonzalezf/org/weather/common/analytics/AnalyticsDataCatalog.java
package rcgonzalezf.org.weather.common.analytics;

public interface AnalyticsDataCatalog {
  interface WeatherListActivity {
    String NO_NETWORK_SEARCH = "NO_NETWORK_SEARCH";
    String MANUAL_SEARCH = "MANUAL_SEARCH";
    String LOCATION_SEARCH = "LOCATION_SEARCH";
    String SEARCH_COMPLETED = "SEARCH_COMPLETED";
  }

  interface SettingsActivity {
    String TEMP_UNITS_TOGGLE = "TEMP_UNITS_TOGGLE";
    String ON_NAME = "ON_NAME";
    String ON_LOAD = "ON_LOAD";
  }
}

File: analytics/src/main/java/rcgonzalezf/org/weather/analytics/analytics/AnalyticsDataCatalog.kt
package rcgonzalezf.org.weather.analytics.analytics

interface AnalyticsDataCatalog {
    interface WeatherListActivity {
        companion object {
            const val NO_NETWORK_SEARCH = "NO_NETWORK_SEARCH"
            const val MANUAL_SEARCH = "MANUAL_SEARCH"
            const val LOCATION_SEARCH = "LOCATION_SEARCH"
            const val SEARCH_COMPLETED = "SEARCH_COMPLETED"
        }
    }

    interface SettingsActivity {
        companion object {
            const val TEMP_UNITS_TOGGLE = "TEMP_UNITS_TOGGLE"
            const val ON_NAME = "ON_NAME"
            const val ON_LOAD = "ON_LOAD"
        }
    }
}
